frontend/src/styles/base/_typography.scss

```diff
@@ -1,6 +1,7 @@
 // Fonts
 @import url('https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@300;400;500;700&display=swap');
 @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
```

Comment:
seems quite wasteful to import a whole font just for the banner, but not sure what the designers' take is on this?
(additionally, maybe we should just import specific weights to reduce pageload times)

Reply:
Yeah, not sure if `wght` is required for the default weight. Got the import statement from [here](https://fonts.google.com/specimen/Lato).